python/paddle/fluid/layers/nn.py

```diff
@@ -7056,10 +7056,10 @@ def image_resize(input,
         out_shape(list|tuple|Variable|None): Output shape of image resize
                                     layer, the shape is (out_h, out_w).
                                     Default: None
-        scale(float|None): The multiplier for the input height or width.
-                         At least one of out_shape or scale must be set.
-                         And out_shape has a higher priority than scale.
-                         Default: None
+        scale(float|None): The multiplier for the input height or width. At
+             least one of :attr:`out_shape` or :attr:`scale`` must be set. 
+             And :attr:`scale` has a higher priority than :attr:`out_shape`. 
```

Comment:
Why `scale` has higher priority? 

Maybe only when `out_shape` is a Variable, need to set `scale`  for the infer-shape?

Reply:
doc here has mistake， out_shape has higher priority than scale in current implement, which is compatible with old version